packages/ts/frontend/src/Connect.ts

```diff
@@ -356,11 +356,10 @@ export class ConnectClient {
       throw new TypeError(`2 arguments required, but got only ${arguments.length}`);
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    const csrfHeaders = globalThis.document ? getCsrfTokenHeadersForEndpointRequest(globalThis.document) : {};
+    const csrfInfo = await csrfInfoSource.get();
```

Comment:
Shouldn't the `Authentication.ts` also use this?

Reply:
Currently, all the security related IT tests are failing as after a successful login, the page refreshed (login view) and shows the login overlay again.